folding-schemes/src/folding/hypernova/decider_eth.rs

```diff
@@ -309,4 +311,201 @@ pub mod tests {
         assert!(verified);
         println!("Decider verify, {:?}", start.elapsed());
     }
+
+    #[test]
+    fn test_decider_serialization() {
```

Comment:
Just not sure on why do we have timers in this test. We can just write formal benchmarks if we want any no?

Reply:
Yup, I just followed what was done with nova, I don't mind removing timers. I agree that we could initiate some benchmarks, but I'm not sure that's on the priority list right now? Wdyt? cc @arnaucube

Reply:
sure, I use to place `dbg!` to know the timings while I test things, but agree with removing them.
Regarding benchmarks, I think that it's not a priority since we're still in the 'exploration phase' where things are not much optimized and we know for sure that are slower than they would be.